docs/developers/wallet.md

```diff
@@ -27,7 +27,7 @@ celestia-appd keys list
 ## Fund a Wallet
 
 For the public celestia address, you can fund the
-previously created wallet via Discord by sending
+previously created wallet via [Discord](https://discord.com/invite/YsnTPcSfWQ) by sending
```

Comment:
Can you please use our global link instead ->  https://discord.gg/celestiacommunity

Reply:
For sure! I'll get that fixed momentarily. I grabbed the original link from the Discord button at the bottom of Celestia's website.